administrator/language/de-DE/plg_system_webauthn.ini

```
@@ -6,42 +6,40 @@
-PLG_SYSTEM_WEBAUTHN_ERR_NOT_DELETED="Der Authentifizierer konnte nicht entfernt werden"
-PLG_SYSTEM_WEBAUTHN_ERR_CREATE_INVALID_LOGIN_REQUEST="Ungültige passwortlose Anmeldeanfrage. Etwas hat nicht funktioniert oder dies ist ein Versuch, die Website zu hacken."
-PLG_SYSTEM_WEBAUTHN_ERR_CANNOT_FIND_USERNAME="Das Feld für den Benutzernamen wurde im Login-Modul nicht gefunden. Die passwortlose Authentifizierung funktioniert auf dieser Website nicht, bitte ein anderes Login-Modul verwenden."
+PLG_SYSTEM_WEBAUTHN_ERR_CREATE_NO_PK="Der Server hat keinen öffentlichen Schlüssel für die Authentifizierer-Registrierung ausgegeben, aber irgendwie eine Authentifizierer-Registrierungsanforderung vom Browser erhalten. Das bedeutet, dass etwas nicht richtig funktioniert hat, bitte erneut probieren."
+PLG_SYSTEM_WEBAUTHN_ERR_CREDENTIAL_ID_ALREADY_IN_USE="Die Anmeldedaten können nicht gespeichert werden. Diese Zugangsdaten werden bereits von einem anderen Benutzer verwendet."
 PLG_SYSTEM_WEBAUTHN_ERR_EMPTY_USERNAME="Bitte nur den Benutzernamen (aber NICHT das Passwort) eingeben, bevor die Anmeldeschaltfläche für die Web-Authentifizierung ausgewählt wird."
 PLG_SYSTEM_WEBAUTHN_ERR_INVALID_USERNAME="Der angegebene Benutzername entspricht nicht einem Benutzerkonto, das eine passwortlose Anmeldung auf dieser Website ermöglicht hat."
+PLG_SYSTEM_WEBAUTHN_ERR_LABEL_NOT_SAVED="Der neue Name konnte nicht gespeichert werden"
+PLG_SYSTEM_WEBAUTHN_ERR_NO_BROWSER_SUPPORT="Leider unterstützt der aktuelle Browser nicht den W3C-Web-Authentifizierungsstandard für passwortlose Anmeldungen oder diese Website wird nicht über HTTPS mit einem gültigen Zertifikat, das von einer Zertifizierungsstelle, welche der Browser vertraut, signiert ist, bereitgestellt. Bitte mit Benutzernamen und Passwort auf dieser Website anmelden."
+PLG_SYSTEM_WEBAUTHN_ERR_NO_STORED_CREDENTIAL="Die gespeicherten Anmeldedaten für diesen Login-Authentifizirer können nicht gefunden werden."
+PLG_SYSTEM_WEBAUTHN_ERR_NOT_DELETED="Der Authentifizierer konnte nicht entfernt werden"
+PLG_SYSTEM_WEBAUTHN_ERR_USER_REMOVED="Der Benutzer für diesen Authentifizierer scheint auf dieser Website nicht mehr zu existieren."
+PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Ermöglicht die Verwaltung passwortloser Anmeldeverfahren unter Verwendung des W3C-Web-Authentifizierungsstandards. Benötigt wird ein unterstützter Browser und Authentifizierer (z.B. Google Chrome oder Firefox mit einem FIDO2-zertifizierten Sicherheitsschlüssel)."
+PLG_SYSTEM_WEBAUTHN_FIELD_LABEL="W3C Web-Authentifizierung (WebAuthn) Anmeldung"
+PLG_SYSTEM_WEBAUTHN_FIELD_N_AUTHENTICATORS_REGISTERED_0="Es wurde noch kein WebAuthn-Authentifizierer eingerichtet"
+PLG_SYSTEM_WEBAUTHN_FIELD_N_AUTHENTICATORS_REGISTERED="%d WebAuthn-Authentifizierer wurden bereits eingerichtet: %s"
+PLG_SYSTEM_WEBAUTHN_FIELD_N_AUTHENTICATORS_REGISTERED_1="WebAuthn-Authentifizierer wurde bereits eingerichtet: %2$s"
```

```suggestion
PLG_SYSTEM_WEBAUTHN_FIELD_N_AUTHENTICATORS_REGISTERED_1="Authentifizierer wurde bereits eingerichtet: %2$s"
```

@zero-24 wir verwenden sonst nur **Authentifizierer**. Mit oder ohne WebAuthn?